scripts/boxd-fork.sh

```
@@ -0,0 +1,672 @@
+  boxd_upload_creds "$vm" || return 1
+  boxd_upload_envs "$vm" || return 1
+  boxd_map_ports "$vm" || return 1
+
+  if [ "$start_tmux" = "1" ]; then
+    # AC#12: tmux + Claude session inside the VM, named claude-<slug>/issue<N>.
+    if "$BOXD_BIN" exec "$vm" -- \
+        "tmux new-session -d -s '$tmux' 'cd langwatch && claude --dangerously-skip-permissions'" \
+        >/dev/null 2>&1; then
+      printf '  started tmux session %s on %s\n' "$tmux" "$vm" >&2
+    else
+      printf '  WARNING: failed to start tmux session %s on %s — start it manually after `boxd connect %s`\n' "$tmux" "$vm" "$vm" >&2
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)